src/node/network_tables.h

```diff
@@ -100,9 +103,12 @@ namespace ccf
     Service service;
     Values values;
     Secrets secrets;
-    Signatures signatures;
     SnapshotEvidence snapshot_evidence;
 
+    // The signatures and serialised_tree tables should always be in sync
```

Comment:
What does in-sync mean here? They both contain a single entry at key 0, and `signatures` is a signature over the root of the entry written to `serialised_tree` at the same version?

Reply:
Good point. I've clarified the comment.